edx_sga/tasks.py

```diff
@@ -61,21 +59,29 @@ def _compress_student_submissions(zip_file_path, block_id, course_id, locator):
     student_submissions = _get_student_submissions(block_id, course_id, locator)
     if not student_submissions:
         return
-    # Build the zip file in memory
-    zip_file_bytes = BytesIO()
-    with zipfile.ZipFile(zip_file_bytes, 'w') as zip_pointer:
-        for student_username, submission_file_path in student_submissions:
-            with default_storage.open(submission_file_path, 'rb') as destination_file:
-                filename_in_zip = '{}_{}'.format(
-                    student_username,
-                    os.path.basename(submission_file_path)
+
+    log.info("Fetched all submission count: %d for zip path: %s ", len(student_submissions), zip_file_path)
```

Comment:
`log.info("Compressing %d student submissions to path: %s ", len(student_submissions), zip_file_path)`